docs/configuration.md

```diff
@@ -0,0 +1,16 @@
+# Configuration
```

Comment:
This should also contain the resource-server specific config stuff from the [server docs](https://github.com/osiam/server/blob/master/docs/detailed_reference_installation.md#configuring-osiam)